server.go

```diff
@@ -28,16 +27,16 @@ func Serve(port int, processor RequestProcessor) {
 	name := processor.GetName()
 	opts := processor.GetOptions() // TODO: figure out command line overrides
 	extproc := &GenericExtProcServer{
-		name: name, 
-		processor: &processor, 
-		options: opts,
+		name:      name,
+		processor: &processor,
+		options:   opts,
 	}
 	epb.RegisterExternalProcessorServer(s, extproc)
 	hpb.RegisterHealthServer(s, &HealthServer{})
 
 	log.Printf("Starting ExtProc(%s) on port %d\n", name, port)
 
-	var gracefulStop = make(chan os.Signal)
+	var gracefulStop = make(chan os.Signal, 1)
```

Comment:
@wrossmorrow here,  accord to the comments from std package:
>
> // Package signal will not block sending to c: the caller must ensure
> // that c has sufficient ***buffer*** space to keep up with the expected
> // signal rate. For a channel used for notification of just one signal value,
// a buffer of size 1 is sufficient.
`